docs/programming-oak.md

```diff
@@ -175,118 +176,92 @@ as a gRPC server:
-```C++
-  // Connect to the Oak Manager.
-  std::unique_ptr<oak::ManagerClient> manager_client =
-      absl::make_unique<oak::ManagerClient>(grpc::CreateChannel(
-          absl::GetFlag(FLAGS_manager_address), grpc::InsecureChannelCredentials()));
+```shell
+./bazel-bin/oak/common/app_config_serializer \
+  --textproto=examples/hello_world/config/config.textproto \
+  --modules=app://examples/target/wasm32-unknown-unknown/release/hello_world.wasm \
+  --output_file=config.bin"
 ```
 <!-- prettier-ignore-end -->
 
-### Starting the Oak Application
-
-To use the `CreateApplication` method, the client needs to specify the
-_application configuration_; this describes the Nodes that make up the
-application, and the initial Node to start. However, for the simple case of a
-one-Node application the `oak::ManagerClient` helper takes care of this; the
-client just need to provide the code that is going to be run in the Node, as a
-Wasm module:
+All these steps are implemented as a part of the
+`./scripts/build_example hello_world`.
 
-<!-- prettier-ignore-start -->
-[embedmd]:# (../examples/rustfmt/client/rustfmt.cc C++ /.*FLAGS_module/ /}/)
-```C++
-  std::string module_bytes = oak::utils::read_file(absl::GetFlag(FLAGS_module));
-  std::unique_ptr<oak::CreateApplicationResponse> create_application_response =
-      manager_client->CreateApplication(module_bytes);
-  if (create_application_response == nullptr) {
-    LOG(QFATAL) << "Failed to create application";
-  }
-```
-<!-- prettier-ignore-end -->
+### Starting the Oak Application
 
-There's also a variant of `CreateApplication` that allows the client code to
-specify a storage provider to connect to, and to control whether logging is
-enabled:
+ISV should run an Oak Application using the Oak Runner:
 
 <!-- prettier-ignore-start -->
-[embedmd]:# (../examples/translator/client/translator.cc C++ /.*FLAGS_module/ /}/)
-```C++
-  std::string module_bytes = oak::utils::read_file(absl::GetFlag(FLAGS_module));
-  std::unique_ptr<oak::CreateApplicationResponse> create_application_response =
-      manager_client->CreateApplication(module_bytes, /* logging= */ true,
-                                        /* storage_address= */ "");
-  if (create_application_response == nullptr) {
-    LOG(QFATAL) << "Failed to create application";
-  }
+```shell
+`./scripts/run_server_asylo --config="$PWD/config.bin"`
 ```
 <!-- prettier-ignore-end -->
 
-The Oak Manager will launch an [Oak Runtime](concepts.md#oak-vm) inside the
+The Oak Runner will launch an [Oak Runtime](concepts.md#oak-vm) inside the
 enclave, and this Runtime will check the provided Wasm module(s) and application
 configuration. Assuming everything is correct (e.g. the Nodes all have a main
 entrypoint and only expect to find the Oak
 [host functions](abi.md#host-functions)), the Oak Runtime opens up a port of its
-own and the `CreateApplication` returns this to the client.
+own and returns this to the caller in an `ApplicationCreationStatus`.
```

Comment:
What is an `ApplicationCreationStatus`, and who's the caller here?

Presumably this contains the host:port that the ISV published to make the Oak App available to the outside world &ndash; be nice to say so.

Reply:
Done